src/jquery.multi-select.js

```diff
@@ -100,55 +130,91 @@
         if (key === escapeKey) {
           _this.menuHide();
         }
-      });
+      })
+      .appendTo(this.$container);
 
-      this.$menu.on('change.multiselect', function() {
-        _this.updateButtonContents();
-      });
```

Comment:
Could this three line removal be a separate commit, as it was fixing a separate bug to this commit.